app/tests/e2e/test_connections.py

```diff
@@ -109,32 +108,13 @@ async def test_delete_connection(
 async def test_bob_and_alice_connect(
     bob_member_client: AsyncClient,
     alice_member_client: AsyncClient,
+    bob_and_alice_connection: BobAliceConnect,
 ):
-    time.sleep(3)
-    invitation_response = await bob_member_client.post(
-        "/generic/connections/create-invitation"
-    )
-    invitation = invitation_response.json()
-
-    accept_response = await alice_member_client.post(
-        "/generic/connections/accept-invitation",
-        json={"invitation": invitation["invitation"]},
-    )
-    alice_connection = accept_response.json()
-
-    bob_connection_id = invitation["connection_id"]
-    alice_connection_id = alice_connection["connection_id"]
-
-    # wait for events to be exchanged
-    # FIXME: listen to webhooks once available
-    time.sleep(5)
-
-    bob_connection = (
-        await bob_member_client.get(f"/generic/connections/{bob_connection_id}")
-    ).json()
-    alice_connection = (
-        await alice_member_client.get(f"/generic/connections/{alice_connection_id}")
-    ).json()
+    time.sleep(2)
```

Comment:
If we write out the test here and use the `assert_webhook_state` method we can wait for the expected webhook event and ditch the sleep (we should avoid those evil sleeps 👿  )

Reply:
agreed